### PR TITLE
[12.0][MIG] purchase_request_product_usage

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

Depends on:

- [x] https://github.com/OCA/purchase-workflow/pull/720

CC @Eficent

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 account-invoicing
 server-tools
 server-ux
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 12.0-mig-purchase_request
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: This is not ready to merge mr. ocabot :robot: :laughing:

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by HviorForgeFlow** [comment]
@lreficent could you update your review? Thanks!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**